data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/BackendVersion.java

```diff
@@ -0,0 +1,25 @@
+package org.opensearch.dataprepper.plugins.sink.opensearch;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public enum BackendVersion {
+    ES6("es_6");
```

Comment:
Thoughts on dropping the "_": `es6`?

Also, we should probably have some explicit options for users to provide for other version.

Perhaps:
* `es6`
* `es7`
* `os1`
* `os2`

How would we like to support newer OS versions as they are added? Maybe a `default` option which aims for the latest? Thus:

* `es6`
* `default`